multiqc/modules/ivar/ivar.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+""" MultiQC module to parse output files from iVar """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+
+from multiqc import config
+from multiqc.plots import bargraph, heatmap
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name = 'iVar',
+            anchor = 'iVar',
+            href = 'https://github.com/andersen-lab/ivar',
+            info = "is a computational package that contains functions broadly useful for viral amplicon-based sequencing."
+        )
+
+        # Find and load iVar trim results
+        self.ivar_data = dict()
+        self.parsed_primers = dict()
+        for f in self.find_log_files('ivar/trim', filehandles=True):
+            parsed_data = self.parse_ivar(f)
+            if parsed_data is not None and len(parsed_data) > 0:
+                self.ivar_data[f['s_name']] = parsed_data
+                self.add_data_source(f, f['s_name'])
```

Comment:
```suggestion
                self.add_data_source(f, section='trimming')
```
No need to add `s_name` if it's the default. As we have more than one data source type, we need to supply the `section` argument though.